pkgs/data/fonts/iosevka/bin.nix

```diff
@@ -1,28 +1,39 @@
-{ stdenv, fetchzip }:
-
+{ stdenv, fetchzip, genericUpdater, common-updater-scripts }:
 let
-  version = "3.4.6";
-in fetchzip {
-  name = "iosevka-bin-${version}";
+  pname = "iosevka-bin";
```

Comment:
What is the purpose of this package, and what does it provide that `iosevka` doesn't? Overall, I'm asking because I'd like to push this towards a minimal PR that only puts `iosevka` into `node-packages.json` to allow @cideM grab it to his next round of `./generate.sh`.

Reply:
According to [the PR that inits it](https://github.com/NixOS/nixpkgs/pull/32106) and [one of the comments it points to](https://github.com/NixOS/nixpkgs/pull/31835#issuecomment-347048242), one of Iosevka's dependencies (otfcc) didn't build on aarch64 back then. However, I no longer see otfcc in the dependencies list (possibly replaced with [ot-builder](https://github.com/ot-builder/monorepo)), so it'd be worth checking if it works now, [which it seems like it does?](https://github.com/NixOS/nixpkgs/pull/99004/checks?check_run_id=1333017601)

Reply:
Oh, there is also [this](https://github.com/NixOS/nixpkgs/pull/101132) now.